turbopack/crates/turbopack-ecmascript/src/lib.rs

```diff
@@ -766,21 +767,14 @@ impl EcmascriptModuleContent {
         specified_module_type: SpecifiedModuleType,
         module_graph: Vc<ModuleGraph>,
         chunking_context: Vc<Box<dyn ChunkingContext>>,
-        references: Vc<ModuleReferences>,
+        (references, esm_references): (Vc<ModuleReferences>, Vc<EsmAssetReferences>),
```

Comment:
Better move all arguments in a options object with `TaskInput`

Reply:
Done